### PR TITLE
Lpeglabel15

### PR DESCRIPTION
**Pull request by sqmedeiros**
Merge this branch into master

### CODE REVIEW

**Review by hugomg — APPROVED** [review]
Looks good to me

File: titan-compiler/lexer.lua [hunk 1]
```diff
@@ -22,7 +22,10 @@ local C, Cb, Cg, Ct, Cmt, Cc = lpeg.C, lpeg.Cb, lpeg.Cg, lpeg.Ct, lpeg.Cmt, lpeg
 local T = lpeg.T
 
 local syntax_errors = require "titan-compiler.syntax_errors"
-local labels = syntax_errors.label_to_int
+local labels = {}
```
Comment (gligneul) [inline titan-compiler/lexer.lua:25]: We don't need this table anymore. Couldn't we just remove this piece of code and use the actual label now?
`s/labels.\([A-Za-z_]*\)/"\1"/g`

**Review by gligneul — DISMISSED** [review]
Looks good, just a minor detail I think we should fix now.

File: titan-compiler/lexer.lua (hunk 1 above)
Comment (hugomg) [reply titan-compiler/lexer.lua:25]: I can fix this in a subsequent pull request. If this is the only problem I think we can merge it now.

**Pull request merged**